build.gradle

```diff
@@ -244,6 +244,7 @@ project(':core') {
   dependencies {
     compile project(':clients')
     compile project(':log4j-appender')
+    compile project(':tools')
```

Comment:
Yes, Its bit odd.  Currently we are building kafka tar build from core sub-project. So we need to add this dependency to bring kafka-tools.jar to libs directory.  We can change this gradle build logic. I will try to fix this issue in next patch.